### PR TITLE
Fix date of 3.1 blog post

### PR DESCRIPTION
**Pull request by ryanto**
Update the date of the 3.1 blog post and add a redirect so the old url still works

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -170,16 +170,6 @@ GEM
       coderay (~> 1.0)
       method_source (~> 0.8)
       slop (~> 3.4)
-    pry (0.9.12.6-i386-mingw32)
```
Comment (MelSumner) [inline Gemfile.lock]: Can you help me understand why you removed these?

Comment (ryanto) [reply Gemfile.lock]: this was the automatic result of a fresh bundle install.

Comment (MelSumner) [reply Gemfile.lock]: okay. Let me pull down this branch first and check that it won't break a windows build :)

Comment (ryanto) [reply Gemfile.lock]: 👍 if it does I can figure out how to get this into the Gemfile so anyone who starts fresh will end up with the correct dependencies. Keep me posted!

**Pull request merged**